public/javascripts/CreateUser.js

```diff
@@ -7,12 +7,8 @@ jQuery('#createUser').submit(function(){
         "lastName":jQuery('#last').val()
     };
     //console.log(toPass);
-    jQuery.ajax({
-        type: "POST",
-        url: window.apiRoute+'/login/createUser/',
-        data: toPass,
-        dataType: 'json',
-        success: function(){
+    navigation.postJSON(window.apiRoute+'/login/createUser/', toPass, function(err, res){
+        if(res) {
             window.alert("Please check your email for an account creation confirmation.");
```

Comment:
As I understand it, Nick is the only one who will be creating users.  This alert should say something like: "An email has been sent to ______ with account confirmation details."